crates/napi/src/tokio_runtime.rs

```diff
@@ -72,6 +72,15 @@ where
   RT.read().unwrap().as_ref().unwrap().block_on(fut)
 }
 
+/// spawn_blocking on the current Tokio runtime.
+pub fn spawn_blocking<F, R>(func: F) -> tokio::task::JoinHandle<R>
```

Comment:
for the sake of tokio-console, it would nice to annotate these as either inline or #[track_caller]